tests/unit/aiplatform/test_automl_forecasting_training_jobs.py

```diff
@@ -257,17 +257,25 @@ def teardown_method(self):
         initializer.global_pool.shutdown(wait=True)
 
     @pytest.mark.parametrize("sync", [True, False])
+    @pytest.mark.parametrize(
+        "training_job",
+        [
+            training_jobs.AutoMLForecastingTrainingJob,
+            training_jobs.SequenceToSequencePlusForecastingTrainingJob,
+        ],
+    )
     def test_run_call_pipeline_service_create(
         self,
         mock_pipeline_service_create,
         mock_pipeline_service_get,
         mock_dataset_time_series,
         mock_model_service_get,
         sync,
+        training_job,
```

Comment:
Move this above sync for consistency's sake.

Reply:
Can you elaborate why `training_job` should be above `sync`? I wanted to keep it aligned with the order `parameterize` was called, so should the order of the calls to `parameterize` be swapped too?

Reply:
Okay I see, feel free to leave it as is.